### ISSUE TITLE
403 Forbidden Error

### ISSUE THREAD

**Issue by RogerWaldrip** [WWBN/AVideo#2645]
Receiving a "403 Forbidden" Error in the browser (on Chrome and Edge) tab when  selecting some videos.  The page displays;

Forbidden
You don't have permission to access /video/45/0064 on this server.

I have uploaded files and they have transcribed (or whatever the term).  I have made the files available.  However, the error seems to only appear when selecting the small sized videos on the web page.  Selecting the large ones at the top of the page doesn't produce the error.

Thinks in advance.  This app looks pretty cool.

**Comment by RogerWaldrip** [comment]
I forgot to mention that you may see the error at;

http://mytubes.thewaldrips.net

**Comment by RogerWaldrip** [comment]
And I meant "encoding" instead of "transcribed".  Silly me.

**Comment by akhilleusuggo** [comment]
looks like you'r running your website from a nextcloud script . Is that correct ? 
https://mytubes.thewaldrips.net/

**Comment by RogerWaldrip** [comment]
I’m not sure how to answer that. Yes, I have a nextcloud/onlyoffice install in a separate virtualhost and the AVideo in another VirtualHost.  The primary or documentroot of the website is a Wordpress site.  It was set up and running several months before I started on the nextcloud/onlyoffice and AVideo pieces.  And, yes, I did get the NextCloud/OnlyOffice up and running before the AVideo.  So I’m not sure how to answer your question as to running my website from a NextCloud script.

I’ve been beating my head against a wall on this one because I am running this all on a QNAP NAS.  I spent the last several days trying to figure out their file hierarchy to make changes to things like log files and what.  Last night (early this morning) I finally figured out that that their system was rewriting my changes to the apache.cong file and setting it back to ‘their’ default version (I thought I was going crazy and either not entering the changes or entering them into an unused example file or something).  So I couldn’t even turn on logging to what I wanted/needed.

OK, back to your question.  I did use a nextcloud script.  Based on what QNAP does, I don’t even know if the nextcloud script made changes to the apache.conf file that even took.

Both the nextcloud/onlyoffice and the AVideo pieces are basically play/learning things for me.  So I’m okay with wiping the nextcloud/onlyoffice and setting up AVideo first then going back to nextcloud/onlyoffice and getting it going later (removing it may not reverse anything anyways).  However, I’m also OK with working with you to resolve it the way it is so that we both have a learning experience that may help people in the future.

I am not running the AVideo on HTTPS, though I don’t have a problem doing that (I don’t have any valid certificates yet) but my NextCloud/OnlyOffice is. Attached are my (QNAP’s) apache config files and the AVideo config file from videos directory, if that will help you any.  I’d more more than happy to give you access to anything I got if that would make it easier on you (anything beyond my base Wordpress site is essentially play/knowledge gathering).  Thanks, and sorry for the long response.

[RogersAvideoConfigFiles.zip](https://github.com/WWBN/AVideo/files/4219917/RogersAvideoConfigFiles.zip)

**Comment by akhilleusuggo** [comment]
@RogerWaldrip Maybe Daniel can help you , I can't . You're running it on Windows .
I've tried it on Windows , I was getting too many errors and many things missing . 
Linux distr are the optimal for this script I believe .

What can you do with your windows machine ? Create a VM ( virtual machine ) .
Download and install Ubuntu 16.04 or 18.04 
https://github.com/WWBN/AVideo/wiki/How-to-install-LAMP,-FFMPEG-and-Git-on-a-fresh-Ubuntu-18.x-for-AVideo-Platform-version-4.x-or-newer

https://tutorials.avideo.com/video/27/how-to-install-youphptube-in-a-fresh-ubuntu-server?channelName=yeshua

**Comment by RogerWaldrip** [comment]
Actually it's not windows but a version of Linux.  I haven't been able to determine from derivative the QNAP QTS OS is from, but appears to be spun off from an old version of Ubuntu.  QNAP has several different CPU types for their NAS's and it appears (though I don;t have any other type other than an Intel CPU version) that they have done a good job of keeping a consistent interface between all of them.

However, they also appear to have made the machines pretty much idiot proof and designed to keep people from mucking up the software, which is probably why it keeps replacing my config files in apache.

I did use a docker version of YouPHPtube but it was something like version 5.  When I did the upgrade or check for upgrade on it it said I was at the newest version.  I abandoned it since it must it so far out of date and I didn't see a way to get it up to the newest version of AVideo.  Maybe I'll go back and pursue that.  Unfortunately, I'm not that versed on Docker stuff yet and the QNAP wants a YAML script to go with any docker containers to be made.

Unless you have another idea, I'll we'll just close this issue and I pursue another avenue.
